src/tools/sfdk/command.cpp

```diff
@@ -1748,12 +1749,22 @@ Worker::ExitStatus BuiltinWorker::setProperties(SetPropertiesTask *task,
         }
         const QString property = assignment.left(splitAt);
         const QString value = assignment.mid(splitAt + 1);
+
+        // CamelCase to snake_case for backward compatibility
+        const QString normalizedProperty = QString(property)
+            .replace(QRegularExpression("([A-Z])"), "-\\1").toLower();
+
         QString errorString;
-        if (!task->prepareSet(property, value, &errorString)) {
+        if (!task->prepareSet(normalizedProperty, value, &errorString)) {
             *exitCode = EXIT_FAILURE;
             qerr() << property << ": " << errorString << endl;
             return NormalExit;
         }
+
+        if (normalizedProperty != property) {
+            qCInfo(sfdk).noquote() << tr("The \"%1\" property is deprecated. Use \"%2\" instead.")
```

Comment:
maybe worth of qCWarning?

Reply:
This is actually less a question of severity and more of the type/source of the event reported https://github.com/sailfishos/sailfish-qtcreator/blob/master/src/tools/sfdk/sfdkglobal.cpp#L69 . I chose to not categorize this event as "Bad usage error" as I feel that category more suitable for fatal errors while the `[I]` gives the clear feedback that the command still succeeded but changes will be needed in future (I am thinking about reading logs from programmatic use of sfdk).